pkg/tasks/stager.go

```diff
@@ -218,18 +218,22 @@ type progress struct {
 }
 
 // runSyncAs runs `s-isync` as the `stagerUser` in a go routine.
-func runSyncAs(ctx context.Context, payload StagerPayload) (chan progress, chan string, *exec.Cmd, error) {
+func runSyncAs(ctx context.Context, payload StagerPayload, concurrency int) (chan progress, chan string, *exec.Cmd, error) {
 
 	tid, ok := asynq.GetTaskID(ctx)
 	if !ok {
 		return nil, nil, nil, fmt.Errorf("invalid context: missing asynq task id")
 	}
 
+	if concurrency == 0 {
```

Comment:
I cannot claim to understand the code to a meaningful degree, but this test seems interesting: if concurrency is 0 (= false?), set it to 1 (= true?). It feels like something is being fixed here that should be set up diffently elsewhere. However, feel free to tell me I'm utterly wrong xD

Reply:
@linusband Let me explain it.

The `concurrency` is an integer referring to the number of concurrent file transfers.  It read in from the configuration YAML file.  If it is not set in the configuration, it will be interpreted by Go as `0` by default.

In this case, I want to take it as no concurrency (and thus set the number to 1, i.e. 1 file transfer at a time).